src/Stryker.Core/Stryker.Core/Reporters/Html/Realtime/RealtimeMutantHandler.cs

```diff
@@ -25,6 +31,28 @@ public void CloseSseEndpoint()
     public void SendMutantTestedEvent(IReadOnlyMutant testedMutant)
     {
         var jsonMutant = new JsonMutant(testedMutant);
+
+        if (_server.HasClientsConnected)
+            SendEvent(jsonMutant);
+        else
+            QueueJsonMutant(jsonMutant);
+    }
+
+    private void SendEvent(JsonMutant jsonMutant)
+    {
         _server.SendEvent(new SseEvent<JsonMutant> { Event = SseEventType.MutantTested, Data = jsonMutant });
     }
+
+    private void QueueJsonMutant(JsonMutant jsonMutant)
+    {
+        _delayedEventQueue.Enqueue(jsonMutant);
+    }
+
+    private void ClientConnectedHandler(object sender, EventArgs e)
+    {
+        while (_delayedEventQueue.Count > 0)
+        {
+            SendEvent(_delayedEventQueue.Dequeue());
```

Comment:
Since we're already saving the events I'm wondering if we might not simply store the events until the end of the mutation testrun so that any client that (re)connects can once again receive all previous results. This way you can close your browser and open it again later. @xandervedder what you do you think

Reply:
That seems like a different problem to me. Here I'm only concerned about waiting for the first client to connect because there could be a delay. This code also only cares about 1 client.

If you want it to replay all previous events when any client connects, we'd have to go about this differently.
You may not want to send all your previous mutants at that point but 1 event that contains all mutant that have been send up to that point. Because there won't be a difference to the user I think.

Reply:
Fair, we can pick that up seperately.